src/connector/src/parser/mod.rs

```diff
@@ -916,12 +920,14 @@ impl SpecificParserConfig {
                     config.topic = get_kafka_topic(props)?.clone();
                     config.client_config = SchemaRegistryAuth::from(props);
```

Comment:
should here be `SchemaRegistryAuth::from(props, &mut options)`? We need both connector name and `format_encode_option` to build a schema registry client